maps/map_files/USS_Almayer/USS_Almayer.dmm

```diff
@@ -68809,7 +68834,9 @@
 	},
 /area/almayer/engineering/upper_engineering)
 "ryR" = (
-/obj/structure/machinery/cm_vending/sorted/uniform_supply/squad_prep,
+/obj/structure/machinery/cm_vending/sorted/uniform_supply/squad_prep{
+	req_access = list(1)
```

Comment:
what is going on here

Reply:
It doesn't unless you've had different experiences testing on local. It worked for SOs when I tested it.